lib/core/kernel.nit

```diff
@@ -644,17 +644,23 @@ universal Byte
 	# `i` bits shift fo the left
 	#
 	#     assert 5u8 << 1    == 10u8
-	fun <<(i: Int): Byte `{ return self << i; `}
+	fun <<(i: Int): Byte is intern do return lsh(i)
+
+	private fun lsh(i: Int): Byte `{ return self << i; `}
```

Comment:
Is this a fallback to support the interpreter? Just curious, could you have written the following?

~~~
fun <<(i: Int): Byte is intern `{ return self << i; `}
~~~